### PR TITLE
Add IndentList and fix some pylint style messages

### PR DESCRIPTION
**Pull request by bc-23**
Add a new class IndentList instead of normal list which takes care of the
required indent syntax.

This is only a change to the style, the result output stays the same.

Signed-off-by: Florian Zavatzki fz@binsec.com

### CODE REVIEW

**Comment by bc-23** [comment]
I signed it!

**Comment by bc-23** [comment]
I signed it!

**Comment by ankenyr** [comment]
Hi @bc-23,

```
Author: removed <removed>
Date:   Tue Oct 4 14:39:27 2016 +0200

    Add IndentList and fix some pylint style messages
    
    Add a new class IndentList instead of normal list which takes care of the
    required indent syntax.
    
    Signed-off-by: removed <removed>

```

I marked the name and email as removed but if you do a `git log` yourself you will see that the email doesn't match what you signed with.

You have a couple options at this point
1) Possibly amend the commit to use the email you signed the CLA with
https://www.atlassian.com/git/tutorials/rewriting-history/git-commit--amend
2) Sign the CLA with the email already used
3) Delete PR, update email, redo commit, make a new push request
https://help.github.com/articles/setting-your-email-in-git/

**Comment by bc-23** [comment]
Hi,

yes, I signed a CLA yesterday for the company, but signing a CLA for a company seems to be more complicated. As far as I can tell it is now pending on google for signing.

When I don't get the CLA today I will update the commit mail address.

File: lib/junipersrx.py [hunk 1]
```diff
@@ -28,6 +28,19 @@
 import logging
 
 
+def junipersrx_list(name, data):
+  return '%s [ %s ];' % (name, ' '.join(data))
+
+
+class IndentList(list):
+  def __init__(self, indent, *args, **kwargs):
```
Comment (ankenyr) [inline lib/junipersrx.py]: Can you insert a line before this def?

Comment (ankenyr) [inline lib/junipersrx.py]: This should also go down under the errors.

**Comment by bc-23** [comment]
I moved the IndentList class below the error definitions and added a newline before the def

File: lib/junipersrx.py (hunk 1 above)
Comment (bc-23) [inline lib/junipersrx.py]: I moved the IndentList class below the error definitions and added a newline before the def

**Pull request merged**